docs/application/dotnet/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,607 @@
+# Component based applications
+
+
+The component based application is one of the Tizen application models. This application model enables you to create an application that provides multiple components in one process.
+In the component based application model, there are three types of components. Each component type has its own lifecycle:
+
+- Service component: The service component has a lifecycle for managing services without UI.
+- Frame component: The frame component has a lifecycle for managing UI resources.
+- Widget component: The widget component has a lifecycle for managing widget resources.
+
+As you create a Tizen project, you can add components to your project. The registered components create instances when the component based application receives a launch request.
+
+The main component based application API features include:
+
+- Application states
+
+  A Tizen native component based application [transitions through various states](#application_states) during its lifecycle.
+
+- Frame component states
+
+  A frame component [transitions through various states](#frame_component_states) during its lifecycle.
+
+- Frame component event callbacks
+
+  The component based application can receive frame component state change events. You can register [callbacks for these events](#frame_component_callbacks) to react to them.
+
+- Service component states
+
+  A service component [transitions through various states](#service_component_states) during its lifecycle.
+
+- Service component event callbacks
+
+  The component based application can receive service component state change events. You can register [callbacks for these events](#service_component_callbacks) to react to them.
+
+- Widget component states
+
+  A widget component [transitions through various states](#widget_component_states) during its lifecycle.
+
+- Widget component event callbacks
+
+  The component based application can receive widget component state change events. You can register [callbacks for these events](#widget_component_callbacks) to react to them.
+
+- Application behavior attributes
+
+  You can [declare components and its specific attributes](#attribute) in the application manifest file.
+
+
+<a name="application_states"></a>
+## Application States
+
+The following figure and table describe the component based application states:
+
+![Running component based applications](./media/application_lifecycle.jpg)
+
+| State        | Description                         |
+|--------------|-------------------------------------|
+| `Ready`      | Application is launched.            |
+| `Create`     | Application starts the main loop.   |
+| `Running`    | Application runs components.        |
+| `Terminate`  | Application is terminated.          |
+
+  > **Note**
```

Comment:
Please update the Note to new style
https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md

Reply:
I applied new style.
But I don't see the new style applied.